### PR TITLE
docs(caching): replace `originalSql` copy and examples with `rollup`

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/752

### CODE REVIEW

File: docs/content/Schema/pre-aggregations.md [hunk 1]
```diff
@@ -470,62 +478,31 @@ cube(`Orders`, {
...
-      primaryKey: true,
-    },
-    created_at: {
-      type: 'time',
-      sql: 'created_at',
-    },
-  },
-});
-```
+Partition tables are refreshed as a whole. When new partition table is available
```
Comment (rpaik) [inline docs/content/Schema/pre-aggregations.md:485]:
```suggestion
Partition tables are refreshed as a whole. When new partition table is available,
```

File: docs/content/Schema/Funnels.md [hunk 2]
```diff
@@ -280,12 +278,10 @@ cube(`PurchaseFunnel`, {
 
   preAggregations: {
     main: {
-      type: `originalSql`
+      type: `rollup`,
```
Comment (keydunov) [inline docs/content/Schema/Funnels.md:281]: This won't work.

File: docs/content/Schema/pre-aggregations.md [hunk 3]
```diff
@@ -285,6 +285,14 @@ Original SQL pre-aggregation is the simplest type of pre-aggregation. As the
 name suggests, it persists the results of the SQL query of the cube in which it
```
Comment (keydunov) [inline docs/content/Schema/pre-aggregations.md:285]: Let's rewrite this section. We need to say that 
1. original SQL pre-aggregations should be used only with internal: true 
2. they do not provide a lot of performance improvements usually
3. they should be used only for `useOriginalSqlPreAggregations` 
4. or for use cases where it is not possible to use rollups

**Review by keydunov — COMMENTED** [review]
Left few comments

**Review by keydunov — APPROVED** [review]

**Comment by keydunov** [comment]
@paveltiunov I'm going to merge it, feel free to take a look and request changes.

**Pull request merged**